pkg/types/powervs/machinepools.go

```diff
@@ -0,0 +1,51 @@
+package powervs
+
+// MachinePool stores the configuration for a machine pool installed on IBM Power VS.
+type MachinePool struct {
+	// VolumeIDs is the list of volumes attached to the instance.
+	//
+	// +optional
+	VolumeIDs []string `json:"volumeIDs"`
```

Comment:
Use `omitempty` as well for optional fields.
```suggestion
	VolumeIDs []string `json:"volumeIDs,omitempty"`
```